index.html

```diff
@@ -612,14 +612,14 @@ <h3>Summary of Members</h3>
         </tfoot>
       </table>
 
-      <p class="note">
-      Other {{WebAppManifest}} members
-        ({{WebAppManifest/scope}},
-        {{WebAppManifest/theme_color}},
-        {{WebAppManifest/related_applications}},
-        {{WebAppManifest/prefer_related_applications}}, and
-        {{WebAppManifest/shortcuts}}) are not currently supported by MiniApp user agents.
-      </p>
+      <div class="note" role="note" id="issue-container-generatedID"><div role="heading" class="note-title marker" id="h-note" aria-level="4"></div><p class="">
+      Other <a data-link-type="idl" data-type="dictionary" href="https://www.w3.org/TR/appmanifest/#dom-webappmanifest"><code>WebAppManifest</code></a> members
```

Comment:
`WebAppManifest` doesn’t exist anymore, we should probably use https://www.w3.org/TR/appmanifest/#dfn-application-manifest

Reply:
Removed the link in https://github.com/w3c/miniapp-manifest/commit/fe1d3f713594aad86aebbb108463afa5f08926a8